kerlescan/rbac_service_interface.py

```diff
@@ -21,9 +21,9 @@ def get_perms(
     ]
     perms = [perm["permission"] for perm in rbac_data]
 
-    if rbac_filters:
+    if type(rbac_filters) == dict:
```

Comment:
`isinstance(rbac_filters, dict)`

Do we expect something else than `dict`?

Reply:
No, we expect only `dict`. But generally we do not have any type control on the params of the method so I do the check here. We can allow it to fail.

And you prompted me to think more about it... and the value could be `None`.
I am adding check now.